### PR TITLE
[danger] various tweaks

### PR DESCRIPTION
**Pull request by dcramer**
- disable security content checks
- disable big pr checks

### CODE REVIEW

**Comment by dcramer** [comment]
pushing the goal to remove more standard noise that we clearly dont care about

File: Dangerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #      and: https://github.com/samdmarshall/pyconfig/blob/develop/Dangerfile
 
 # set the number of lines that must be changed before this classifies as a "Big PR"
-@S_BIG_PR_LINES ||= 500
+@S_BIG_PR_LINES ||= 0
```
Comment (mattrobenolt) [inline Dangerfile]: I think this is valid imo. I like this, otherwise we'll go back to never updating changelog.

Comment (dcramer) [inline Dangerfile]: this is just the check that says "have you considered splitting up this PR?:"

Comment (mattrobenolt) [inline Dangerfile]: Oh, derp. Yeah, that one is silly.

**Review by mattrobenolt — APPROVED** [review]

**Pull request merged**